luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -112,10 +125,30 @@ public KeyValueStore getKeyValueStore(String keyValueStoreName) throws Unsupport
         }
     }
 
+    /**
+     * Retrieve or build a PhysicalTable.
+     *
+     * @param tableName the name for the PhysicalTable to be retrieved or built.
+     *
+     * @return the dimension instance corresponding to this name.
```

Comment:
Should be `PhysicalTable` instance, not `dimension` instance.

Reply:
I only realized it and fixed it in the Logical table implementation. I can change it here.